.github/workflows/main.yml

```diff
@@ -152,12 +152,11 @@ jobs:
           name: diff
           path: diff
       - name: Update DNSLink (if on the main branch)
-        run: npx dnslink-dnsimple --domain dist.ipfs.io --link /ipfs/${{ steps.cid-reader.outputs.CID }}
+        run: npx dnslink-dnsimple --domain dist.ipfs.tech --link /ipfs/${{ steps.cid-reader.outputs.CID }}
```

Comment:
.io will be CNAME'd  to .tech when https://github.com/protocol/infra/pull/918 is merged